### DISCUSSION TITLE
240318

### DISCUSSION THREAD

**Discussion by jisunipark** [codeit-part4-8team-project/main#54] [팀회의]
## 멘토님 질문 사항

**Comment by caseV12** [comment]
## 논의사항

## 어제 한 일
x
## 오늘 할 일
소셜 로그인 로직이 바뀌어서 새로 작성해야 합니다(방금 채린님이 말씀해주신 부분)
useAxios 수정
## 오늘의 일정 (코어 타임)
x
## 멘토님 질문 사항

**Comment by jisunipark** [comment]
## 논의사항
- 각자 하루에 두 번 정도는 팀 레포 PR 확인하기로 해서 12시간 이내에는 머지될 수 있도록 하면 좋을 것 같습니다.
- 짧은 코멘트라도 각자 1PR 1리뷰 단다고 생각하면 좋을 것 같습니다.
- 깃허브 앱 다운 받아서 알림 활성화 해놓으면 팀 레포에 새롭게 올라오는 PR이나 코멘트를 놓치는 일이 줄어들 것 같습니다.

## 어제 한 일
- React-hook-form 공부

## 오늘 할 일
- 

## 오늘의 일정 (코어 타임)
- x

## 멘토님 질문 사항

**Comment by MoonPillGyeom** [comment]
## 논의사항
x
## 어제 한 일
제어컴포넌트 해결

## 오늘 할 일
추가된 ui 추가작업하고 input에 훅폼 물리기,
링크가 3개가 있는데 각 로고 클릭시에 각각의 링크 이름이 나오게끔 만들기

## 오늘의 일정 (코어 타임)
x
## 멘토님 질문 사항
x
